appveyor.yml

```diff
@@ -1,7 +1,6 @@
 version: 1.0.{build}-{branch}
 image:
   - Visual Studio 2017
-  - Visual Studio 2017 Preview
```

Comment:
Remove Preview, appveyor are too slow when doing CI.

Reply:
I'd remove 2015, since it is persistently failing and no one seems to know how to fix it.

Reply:
2015 is fixed:) about mingw. Anyway, remove is also a good idea, I don't know where is  the mingw on Visual Studio 2017

Reply:
2015 also  removed.